Frontend/research/src/Actions/Index.js

```diff
@@ -79,6 +79,7 @@ export const getCategory = () => dispatch => {
         .then(res => {
             console.log(res);
             dispatch({ type: GET_LINK_SUCCESS, payload: res.data.data })
+            setTimeout(function(){window.location.reload();},300);
```

Comment:
try at all costs to avoid this line of code in a react app. you should not ever need to refresh the app to get your data to rerender appropriately and if you do, there is a logic adjustment that you can make somewhere to remedy needing a reload. when you reload the app, any data you had in your store is lost, its like the user has just opened the app